### PR TITLE
Add an `api-version` field for `BulkMetadataResponse`

### PR DESCRIPTION
**Pull request by nahteb**
We want versioning for the response object as well as the API metadata,
so that we can manage changes to it.

### CODE REVIEW

File: schemas/v1alpha/bulk-metadata-response.json [hunk 1]
```diff
@@ -5,9 +5,17 @@
   "description": "A JSON Schema to manage the format of API response objects",
   "type": "object",
   "required": [
+    "api-version",
     "apis"
   ],
   "properties": {
+    "api-version": {
```
Comment (jamietanna) [inline schemas/v1alpha/bulk-metadata-response.json:12]: It's only the second duplication, so happy to leave it off for now, but did we want to make this its own object so we can just `$ref` it in these schemas?

**Review by jamietanna — COMMENTED** [review]
Can the renames please be their own commit?

File: schemas/v1alpha/bulk-metadata-response.json (hunk 1 above)
Comment (nahteb) [reply schemas/v1alpha/bulk-metadata-response.json:12]: good idea, I'll wait till it happens for a third time and do it then

File: examples/java/spring-boot/src/test/java/uk/gov/api/springboot/models/metadata/BulkMetadataResponseTest.java [hunk 2]
```diff
@@ -62,12 +62,24 @@ void apisIsNullWhenResponseIsNull() throws IOException {
...
 
-      jsonContent = jacksonTester.write(BulkMetadataResponse);
+      jsonContent = jacksonTester.write(bulkMetadataResponse);
 
       assertThat(jsonContent).extractingJsonPathArrayValue("apis").isEmpty();
     }
+
+    @Test
+    void apiVersionIsSerialized() throws IOException {
+      BulkMetadataResponse bulkMetadataResponse = new BulkMetadataResponse();
```
Comment (jamietanna) [inline examples/java/spring-boot/src/test/java/uk/gov/api/springboot/models/metadata/BulkMetadataResponseTest.java:74]: one for a follow-up, but I'd say that this is a bit too verbose, let's instead aim for:

```suggestion
      BulkMetadataResponse response = new BulkMetadataResponse();
```

(but also let's use `var`)

**Review by jamietanna — APPROVED** [review]

File: examples/java/spring-boot/src/test/java/uk/gov/api/springboot/models/metadata/BulkMetadataResponseTest.java (hunk 2 above)
Comment (nahteb) [reply examples/java/spring-boot/src/test/java/uk/gov/api/springboot/models/metadata/BulkMetadataResponseTest.java:74]: will do!

**Pull request merged**